fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -595,7 +598,6 @@ private void checkForInsufficientStorageException(final Throwable rootThrowable,
      * @return 204 No Content (for updated resources), 201 Created (for created resources) including the resource
      *    URI or content depending on Prefer headers.
      */
-    @SuppressWarnings("resource")
```

Comment:
With this annotation removed, there appears to be a new warning of a potential resource leak.

Reply:
Sorry, Eclipse was complaining about this not being needed. I'll just leave these sorts of things alone in future :-)